### DISCUSSION TITLE
When could users choose server location?

### DISCUSSION THREAD

**Discussion by panism** [getlantern/lantern-forum-cn#547] [General]
need to use openai and claude on a daily basis, please fix the problem asap.
cheers

**Comment by MaxMKMax** [comment]
Hi @panism , thanks for your feedback. We are appreciate your usage and concern ingabout this features. it would be very soon. Actually we had discuss this for several times, developer are concerning about the feature request from users. They are on the way building it. Hope it will be ready to release soon. We will update you once it is ready.

Thanks for choosing Lantern and using our software. We are happy to have users like you.
